frontend/packages/gitops-plugin/console-extensions.json

```diff
@@ -9,7 +9,6 @@
       "insertBefore": "helm",
       "name": "%gitops-plugin~Environments%",
       "href": "/environments",
-      "namespaced": true,
```

Comment:
is it not namespaced scoped?

and if it's based on gitops operator can't we have it feature flagged?

cc @divyanshiGupta

Reply:
@invincibleJai no, it shouldn't be namespace scoped, confirmed it with @reginapizza 
And, it is already feature flagged

Reply:
Thanks for the update, missed seeing the feature flag with `OPENSHIFT_GITOPS`